src/ducks/Studio/timeseries/hooks.js

```diff
@@ -3,6 +3,7 @@ import { client } from '../../../index'
 import { getQuery, getPreTransform } from './fetcher'
 import { normalizeDatetimes } from './utils'
 import { mergeTimeseriesByKey } from '../../../utils/utils'
+import {Metrics} from "../../SANCharts/data";
```

Comment:
why from SANCharts? new place - https://github.com/santiment/app/blob/master/src/ducks/dataHub/metrics/index.js#L81

Reply:
Could you pls remove `;` at the end of row

Reply:
IDEA :(

Replaced

Reply:
change settings in IDEA for this case)

Reply:
![image](https://user-images.githubusercontent.com/522287/80588509-e9470e00-8a31-11ea-815e-bb95603b5b9f.png)
Honestly also "Metrics" should have space around.

Reply:
Command 'yarn format' was removed 2 weeks ago. Returned.